src/event.rs

```diff
@@ -222,6 +222,18 @@ impl CGEvent {
             CGEventGetFlags(self.as_concrete_TypeRef())
         }
     }
+
+    pub fn set_string_from_utf16(&self, buf: &[u16]) {
```

Comment:
Depends on the implementation of `CGEventKeyboardSetUnicodeString`. I tested with the examples found [here](https://github.com/rust-lang/rust/blob/4379e2fa08bf31125efa1a9d8bc3c85b16e2c2f3/src/libcollectionstest/string.rs#L144) and it looks like it doesn't output anything for invalid strings.

Perhaps it would be better to instead call it `set_string_from_utf16_unchecked`?